pypyr/dsl.py

```diff
@@ -210,6 +210,7 @@ def __init__(self, step):
                 # of course, it might not be a string. in line with duck
                 # typing, beg forgiveness later. as long as it loads
                 # the module, happy days.
+                logger.notice(step)
```

Comment:
remove, pls. . . want to keep the new NOTIFY level exclusively for when operator explicitly asks for it via either `.echo` or `description`.

Reply:
yes this makes sense